package.json

```diff
@@ -56,6 +59,7 @@
   "dependencies": {
     "@sentry/node": "4.6.4",
     "aws-sdk": "2.377.0",
+    "bash-color": "^0.0.4",
```

Comment:
I don't think it matters a whole lot because this is a server-only project right now, but I would expect this to be in `devDependencies`, not `dependencies`.

Reply:
good catch, making the changes!